cfme/tests/test_login.py

```diff
@@ -12,6 +12,35 @@
 pytestmark = pytest.mark.usefixtures('browser')
 
 
+def new_cred(username, password):
+    username = username or conf.credentials['default']['username']
+    password = password or conf.credentials['default']['password']
+    cred = Credential(principal=username, secret=password)
+    return cred
+
+
+def new_user_instantiate(appliance, username=None, password=None, name=None):
+    cred = new_cred(username=username, password=password)
+    name = name or 'Administrator'
+    user = appliance.collections.users.instantiate(credential=cred, name=name)
+    return user
+
+
+def new_user_create(appliance, request, username=None, password=None, group=None, name=None):
```

Comment:
This is acting like a fixture, by adding the finalizer, personally I think this should be a fixture this actually then makes the test focus solely on updating the password and not on creating the user.